mne/utils/__init__.pyi

```diff
@@ -381,4 +382,5 @@ from .numerics import (
     split_list,
     sum_squared,
 )
+from .numpy import numpy_fromfile
```

Comment:
[nitpick] I don't love using the name of another prominent module as a submodule name within MNE. Would it make sense to put this in `fixes`? If not, let's rename the new file to... `blob_io`?  I hate that name but I'm in a rush and it's the first thing I thought of